aiida_quantumespresso/parsers/parse_raw/pw.py

```diff
@@ -366,6 +371,22 @@ def parse_stdout(stdout, input_parameters, parser_options=None, parsed_xml=None)
                         parsed_data['dipole' + units_suffix] = default_dipole_units
                         break
 
+            # saving the SCF convergence accuracy for each SCF cycle
+            # this is not always present in the block starting with 'iteration #'
+            # so the array might be of different size for each relax cycle
+            elif 'estimated scf accuracy' in line:
+                try:
+                    value = float(line.split()[-2])* ry_to_ev
+                    scf_index += 1
```

Comment:
Instead of keep track of the `scf_index` through a variable and upping it in `scf_accuracy_index`, wouldn't it be nicer to just at the end of the frame, do something like:
```
current_index = len(trajectory_data.get('scf_accuracy', [])
trajectory_data.setdefault('scf_accuracy_index', []).append(current_index)
```
```

Reply:
Hi,

Your suggestion is an option, but in the way the code is now it needs to be at the beginning of the frame, and ``trajectory_data['scf_accuracy']`` does not before the first assignment. 

This would also change it so that the array starts from 0 and does not contain the last index (the length of the array). It could actually be an alternative.

Also, right now the ``scf_accuracy`` entry is not in the output unless it is assigned at least once, but we might want to pass an empty array anyway? Not sure.

Reply:
If you want to keep the index array to keep the starting index of each frame, then you can already create `trajectory_data['scf_accuracy_index'] = [0]` before the loop over the frames starts. Then at the end of each frame you can call `trajectory_data['scf_accuracy_index'].append(current_index)`

Reply:
I see what you mean now. I will change it, thanks for the suggestion